src/Conversion/ONNXToKrnl/Math/Reduction.cpp

```diff
@@ -370,9 +377,15 @@ struct ONNXReduceSumOpLowering : public ConversionPattern {
     int64_t outRank = memRefOutType.getRank();
 
     // KeepDims
-    auto keepdims = llvm::dyn_cast<ONNXReduceSumOp>(op).keepdims();
+    auto keepdims = reduceSumOp.keepdims();
```

Comment:
[Suggestion]: use the actual type rather than auto here (because the static type is not immediately obvious).